Redesign HUDnav

Heartfyre opened the pull request:
![hudnav3](https://user-images.githubusercontent.com/20871500/41833674-a078e39a-7851-11e8-88fc-89ca6c813062.gif)

Replaced second retweet icon with refresh icon as well.

jamkoo commented:
Hey, I like the flag concept. but having it repeat multiple times when it's opened might not be very good. Maybe have the flag horizontal and grow out longer if you open it? 

or try using the name bg texture so it's similar to the other UI. :)

Heartfyre commented:
I reworked the hudnav again and added a new style to it. Please check it out and let me know, what you think of it. :)

![hudnav](https://user-images.githubusercontent.com/20871500/41912475-34ba01a2-7950-11e8-9e7c-f1398ec1ab19.gif)

Ajackster commented:
Thanks! This has been merged [here](d9721c00d9123db405e44ea4ea6a7eb288f82690)
